### ISSUE TITLE
Multiple Account Support

### ISSUE THREAD

**Issue by josephwegner** [waltzio/waltz#205]
Obviously, we need to start thinking about this.

I think that from a technical point of view, this isn't actually all that hard.  We just need to start storing arrays of credentials for each site, rather than a single object.  The bigger trick, I think, is UX - how do we ask the user which account to sign in with?

The trickiest part will be working with sites like Google that allow multiple concurrent sessions.  I am 100% OK with pushing that, and just getting a basic multiple account system rolled out for now.

**Comment by jessepollak** [comment]
I was thinking this issue was something else: supporting multiple Clef accounts on the same computer, which I think is something we do need to do, but can hold off on for a little longer.

Multiple web account support is definitely a feature that I'd like: I use multiple accounts on Gmail, HelpScout and a bunch of other sites — being able to easily choose the account I want would be great.

**Comment by josephwegner** [comment]
@jessepollak In order to do multiple Clef accounts, we need to finish #14, I believe.  I also think that's an important feature.

**Comment by jessepollak** [comment]
Yeah, for sure. I can look into this in the new year. I don't think the
name spacing will be that hard, but there are some tricky UX issues I
think.

**Comment by My1** [comment]
has there been anything to this yet? sounds intresting
